src/parser/paladin/holy/modules/talents/RuleOfLaw.js

```diff
@@ -63,7 +63,6 @@ class RuleOfLaw extends Analyzer {
               uptimeHistory={history}
               start={this.owner.fight.start_time}
               end={this.owner.fight.end_time}
-              style={{ height: '100%' }}
             />
           </div>
```

Comment:
Shouldn't holy paladins stay untouched in this PR, so style should still be usable in the uptimebar?

Reply:
I somewhat disagree. The UptimeBar doesn't _look right_ when `height` is not 100%. I'm sure that at some point there was a valid use case for it, but instead of fixing css that will end up not being used, I'd much rather enforce it to a sane value.

I can remove the change from the holy pal module though, but IMHO it's just keeping dead code there for the sake of it at this point.

Reply:
Hence adding the default value but letting others change it if they want to

Reply:
Done in 088af66c699795a7009fc633bc8449658bce0940